config/webpack/serve.webpack.config.js

```diff
@@ -80,14 +81,24 @@ function getWebpackConfig(skyPagesConfig) {
         key: fs.readFileSync(path.join(__dirname, '../../ssl/server.key')),
         cert: fs.readFileSync(path.join(__dirname, '../../ssl/server.crt'))
       },
-      publicPath: common.appConfig.base
+      publicPath: skyPagesConfigUtil.getAppBase(skyPagesConfig)
     },
-    debug: true,
     devtool: 'cheap-module-eval-source-map',
     plugins: [
       new NamedModulesPlugin(),
-      WebpackPluginDone
-    ]
+      WebpackPluginDone,
+      new LoaderOptionsPlugin({
+        debug: true
+      })
+    ],
+    node: {
```

Comment:
Perhaps we could try leaving this out?  The concept makes sense as it changes what's exposed, but I've never seen great documentation on this section.  I'm curious to see if it's truly necessary.